src/openrct2/network/NetworkTypes.h

```diff
@@ -126,10 +126,11 @@ template<typename T, size_t _TypeID> struct NetworkObjectId_t
 };
 #pragma pack(pop)
 
+using NetworkRideId_t = ride_id_t;
```

Comment:
Won't this mean that there is a specialisation in the data serialiser for ride_id_t?